qiskit/synthesis/clifford/clifford_decompose_layers.py

```diff
@@ -32,24 +37,22 @@
 )
 
 
-def _default_cx_synth_func(mat, validate):
```

Comment:
Is there a reason for removing the `validate` argument?

Reply:
Yes, since generally, the synthesis functions such as `synth_cnot_depth_line_kms` or `synth_cnot_count_full_pmh` does not have a `validate` argument. I will just remove the validations, since they are not needed.